articles/tokens/access-token.md

```diff
@@ -0,0 +1,144 @@
+```
+
+The token in this example decodes to the following claims:
+
+```json
+{
+  "alg": "RS256",
+  "typ": "JWT"
+}
+.
+{
+  "iss": "https://example.auth0.com/",
+  "aud": "https://api.example.com/calandar/v1/",
+  "sub": "usr_123",
+  "scope": "read write",
+  "iat": 1458785796,
+  "exp": 1458872196
+}
+```
+
+Before permitting access to the API using this token, the API must do the following:
+
+1. Ensure that the token is intended to be used at the API by checking that the value of `aud` is identical to the API's identifier.
+1. Ensure that the token has not expired by comparing the value of `exp` to the current time.
+1. Ensure that the token was issued by a trusted authorization server. In this case, Auth0 is the trusted authorization server and a secret, `keyboardcat`, is known only to Auth0 and the API. The signature of the token is validated using this secret.
+1. Ensure that the token has the correct scopes to perform the requested operation.
+
+If any of these check fail, the token is invalid and the request should be rejected.
+
+Once these checks have been performed successfully, the API can be assured that:
+
+- The token was issued by Auth0.
+- The token was issued to an application being operated by the user with an identifier of `usr_123`.
+- The user granted the application access to read and write his or her calendar.
+
+The API can now process the request, allowing the application to read and write to user `usr_123`'s calendar.
+
+## Lifetime
+
+By default, access tokens are valid for 24 hours. The token lifetime can be controlled on a per-API basis. The validity period can be increased or decreased based on the security requirements of each API.
+
+Once expired, an access token can no longer be used to access an API. In order to obtain access again, a new access token needs to be obtained. This can be done by repeating the OAuth flow used to obtain the initial access token.
+
+In some situations, it is desirable to have permanent, ongoing access to an API without having to repeat an OAuth flow. This is often referred to as `offline access`, and is possible with the use of a [refresh token](/tokens/refresh-token).
+
+A refresh token is issued from the OAuth 2.0 endpoints along with the access token. When the access token expires, the refresh token can be used to obtain a fresh access token with the same permissions, without further involvement from a user. Note that offline access is enabled as a policy of the API the access token grants access to. If the API does not permit offline access, a refresh token will not be issued. In such circumstances, the OAuth flow must be repeated in order to obtain a new access token.
+
+For more information on refresh tokens and how to use them refer to: [Refresh Token](/tokens/refresh-token).
+
+## Revoke access
+
+Revoking access is not supported at the moment.
```

Comment:
Worth mentioning that the best mechanism to revoke access is the token expiration itself. Customer should choose a sensible expiration time according to their domain. e.g. a banking API might have shorter lived token vs a TODO list API.